src/style.css

```diff
@@ -1,3 +1,66 @@
 body {
   margin: 0;
+  padding: 0;
+  font-family: Roboto, serif;
+  font-size: 12px;
+  font-weight: 500;
+  text-underline-position: from-font;
+  text-decoration-skip-ink: none;
+}
+
+.header {
+  display: flex;
+  flex-direction: row;
+  height: 60px;
+  justify-content: space-between;
+  padding: 0 50px;
+  align-items: baseline;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  margin: auto 0;
+}
+
+.navbar {
+  margin: auto 0;
+  display: flex;
+  line-height: 60px;
+}
+
+.nav-link {
+  text-decoration: none;
+  color: black;
```

Comment:
Ensure that the color contrast between the text and background is sufficient for accessibility. The current color `black` on a potentially light background should be checked against accessibility standards.